fe/src/pages/Home/HomeMain/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { $Template } from '@styles/PageTemplate.style';
+import { $ListItemContainer } from '@pages/Home/HomeMain/HomeMain.style';
+import Navbar from '@components/molecules/Navbar';
+import Dropdown from '@molecules/Dropdown';
+import Icon from '@atoms/Icon';
+import ListItem from '@molecules/ListItem';
+import MainTabBar from '@molecules/TabBars/MainTabBar';
+import ConvertPriceFormat from '@utils/convertPriceFormat';
+
+import { mockData, mockUserTowns } from './mockData';
+
+const HomeMain = () => {
+  return (
+    <$Template>
+      <Navbar>
+        <Dropdown towns={mockUserTowns} />
+        <button>
+          <Icon name="category" />
+        </button>
+      </Navbar>
+      <$ListItemContainer>
+        {mockData.data.products.map(product => (
+          <ListItem
+            key={product.productId}
+            productId={product.productId}
+            title={product.title}
+            town={product.town}
+            price={ConvertPriceFormat(product.price)}
+            status={product.status}
+            createdAt={product.createdAt}
+            countInfo={product.countInfo}
+            imgUrl={product.imgUrl}
+            isCurrentUserItem={true}
+            onItemClick={() => console.log('onItemClick')}
+          />
```

Comment:
map의 인자로 넘어오는 product를 모두 사용해야 한다면 product를 통째로 넘겨주면 코드 작성할 때 더 편리합니다!

```suggestion
        {mockData.data.products.map(product => (
          <ListItem
            {...product}
            key={product.productId}
            isCurrentUserItem={true}
            onItemClick={() => console.log('onItemClick')}
          />
```

또는

```suggestion
        {mockData.data.products.map(product => (
          <ListItem
            key={product.productId}
            product={product} // ListItem 컴포넌트 내부에서 distructuring 해서 사용
            isCurrentUserItem={true}
            onItemClick={() => console.log('onItemClick')}
          />
```

Reply:
🥲 정말 이럴 때, 아직 기본기가 많이 부족하구나 느낍니당 ㅠㅠ ㅋㅋㅋㅋㅋㅋㅋㅋ 제안 주신 코드 보자마자, `왜 이걸 생각 못했을까?!` 했어요ㅋㅋㅋㅋㅋㅋ 진짜 당장 수정해버리겠습니닼ㅋㅋ 감사합니다, 어텀~!!! 🥳